packages/tools/changelog-generator-wrapper/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@fluid-private/changelog-generator-wrapper",
-	"version": "1.0.0",
+	"version": "2.1.0",
 	"private": true,
 	"homepage": "https://fluidframework.com",
 	"repository": {
```

Comment:
directory (line 9) needs updated to new location

Reply:
Good catch; I also opened #21605 to catch and fix these moving forward.